integration-test/trace-recorder/src/main/resources/integrationTestData/TraceDebuggerExamples/org.examples.integration.loops.EmptyForLoopRepresentationTest_operation.txt

```diff
@@ -1,6 +1,7 @@
 # Thread 1
 EmptyForLoopRepresentationTest.operation() at :0
   escape = "START" at EmptyForLoopRepresentationTest.kt:19
+  StringBuilder.<init>(): "" at EmptyForLoopRepresentationTest.kt:20
```

Comment:
The current notation looks too technical for me. Maybe `new StringBuilder()` instead?